lib/ci-stack.ts

```diff
@@ -80,7 +85,13 @@ export class CIStack extends Stack {
       oidcCredArn: importedOidcConfigValuesSecretBucketValue.toString(),
       useSsl,
       runWithOidc,
-    });
+    },
+    {
+      agentNodeSecurityGroup: securityGroups.agentNodeSG.securityGroupId.toString(),
```

Comment:
Seems like the agent configuration should be passed into the CloudAgentNodeConfig object instead of into the jenkins main node.

Reply:
Done